### ISSUE TITLE
2.2.6 RC7 broke running of files as admin

### ISSUE THREAD

**Issue by JayanWarden** [openzfsonwindows/openzfs#404]
### System information
zfswin-2.2.6rc7
zfs-kmod-zfswin-2.2.6rc7
Windows 11 23H2

### Describe the problem you're observing

Upon updating to the new RC, applications cannot be started as Admin on a ZFS volume anymore

### Describe how to reproduce the problem

Create a simple dummy .bat file on a ZFS volume.
In my testing, I used a ZVOL directly, not a dataset.

Then, try to launch the .bat file with rightclick -> run as admin.

The command will fail with "Auf [test.bat] konnte nicht zugegriffen werden" , translating to "Could not access [test.bat]".

The .bat file can be launched without admin without problems.

**Comment by lundman** [comment]
Huh that is surprising. I'll check into it

**Comment by lundman** [comment]
Yeah definitely doesn't work. Not much in FileSpy output either - peculiar.

**Comment by lundman** [comment]
You can change the error by setting mimic on, so that is interesting. Launching as admin cares about filesystem type?

**Comment by JayanWarden** [comment]
At least on 2.2.3-RC6 this still worked. I updated from that release, so something might have changed

**Comment by lundman** [comment]
I tried 2.2.6-rc6, rc5, rc4 and had the same issue. But didn't go as far back as 2.2.3. But it'll just be a quirk somewhere.

**Comment by lundman** [comment]
da1fdfb plus setting mimic=ntfs will fix it.

**Comment by JayanWarden** [comment]
I will test once a new RC gets pushed.

mimic is a parameter for zpool import ?

**Comment by lundman** [comment]
It's for any dataset, like `zfs set com.apple.mimic=ntfs POOL/dataset`

**Comment by sskras** [comment]
ping @JayanWarden :)
